### PR TITLE
Implement pseudo-property `import.meta`

### PR DESCRIPTION
**Pull request by jedel1043**
This Pull Request implements the `import.meta`  pseudo-property.

### CODE REVIEW

**Review by raskad — APPROVED** [review]

File: boa_engine/src/vm/opcode/meta/mod.rs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+/// Stack:
+///  - **=>** `import.meta`
+#[derive(Debug, Clone, Copy)]
+pub(crate) struct ImportMeta;
+
+impl Operation for ImportMeta {
+    const NAME: &'static str = "ImportMeta";
+    const INSTRUCTION: &'static str = "INST - ImportMeta";
+
+    fn execute(context: &mut Context<'_>) -> JsResult<CompletionType> {
```
Comment (HalidOdat) [inline boa_engine/src/vm/opcode/meta/mod.rs:53]: Would be nice to have a link the the spec from where the steps where taken.

File: boa_engine/src/vm/opcode/meta/mod.rs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+        // 3. Let importMeta be module.[[ImportMeta]].
+        // 4. If importMeta is empty, then
+        // 5. Else,
+        //     a. Assert: importMeta is an Object.
+        let import_meta = src
+            .import_meta()
+            .borrow_mut()
+            .get_or_insert_with(|| {
+                //     a. Set importMeta to OrdinaryObjectCreate(null).
```
Comment (HalidOdat) [inline boa_engine/src/vm/opcode/meta/mod.rs:72]:
```suggestion
                // a. Set importMeta to OrdinaryObjectCreate(null).
```
Could we remove the unneeded indentation

File: boa_engine/src/vm/opcode/meta/mod.rs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+/// `ImportMeta` implements the Opcode Operation for `Opcode::ImportMeta`
+///
+/// Operation:
+///  - Push the current `import.meta` to the stack
+///
+/// Operands:
+///
+/// Stack:
+///  - **=>** `import.meta`
```
Comment (HalidOdat) [inline boa_engine/src/vm/opcode/meta/mod.rs:45]:
```suggestion
```
Maybe we should omit this, to be consistent with other Opcode implementations. This info is also on the `Opcode` enum too.

**Review by HalidOdat — COMMENTED** [review]
Looks good to me! Just some small nitpicks :)

**Pull request merged**